cx_Freeze/macdist.py

```diff
@@ -154,10 +153,19 @@ def setRelativeReferencePaths(self):
             references = otool.stdout.readlines()[1:]
 
             for reference in references:
-
                 # find the actual referenced file name
                 referencedFile = reference.decode().strip().split()[0]
 
+                # sometimes otool seems to have strange output for  referenced
+                # files, like
+                # ".../Contents/MacOS/lib/numpy/core/lib/libnpymath.a(npy_math.o):"
+                # and the file can not be located (in this case the
+                # libnpymath.a references itself)
+                if not os.path.exists(referencedFile):
+                    print("ERROR: file {} does not exist. Skipping it".format(
+                        referencedFile))
+                    continue
+
                 if referencedFile.startswith('@executable_path'):
```

Comment:
If the referencedFile starts with @executable_path, is the code you just added going to cause it to get skipped? Perhaps your code block is in the wrong place?

Reply:
Indeed if the referencedFile starts with '@executable_path', it was skipped any way. But it will be better to not have the 'ERROR' print for this case, so the block quote should be moved after the '@executable_path' test.